src/cml.js

```diff
@@ -240,40 +241,50 @@ class CML {
   }
 
   async pr_create(opts = {}) {
-    const { globs = ['dvc.lock', '.gitignore'], md } = opts;
+    const {
+      git_remote: remote = 'origin',
+      git_user_email: user_email = 'olivaw@iterative.ai',
+      gir_user_name: user_name = 'iterative-olivaw',
```

Comment:
do we need to re-define these constant all over again? These are always going to be defined for this function so they're args, not opts?

Reply:
Agree. It should be derived from CML.js package